src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1385,7 +1384,7 @@ internal void SaveAs(string path)
         {
             try
             {
-                Model.CurrentWorkspace.SaveAs(path, EngineController.LiveRunnerRuntimeCore);
+                model.SaveAs(path);
```

Comment:
CustomNodes use workspace.SaveAs which is still the old XML save, I think it is not saving properly for now. I will investigate how to handle all cases.